ultravox/training/configs/llama3_whisper_s_cont_librispeech_960.yaml

```diff
@@ -0,0 +1,33 @@
+exp_name: "llama3_whisper_s_cont_librispeech_960"
```

Comment:
For now we have kept the configs directory very minimal.

In general my assumption is that configs in `main` should not be experimental.
IMO, PRs that want to merge configs should have different criteria that are based on comparing metrics not just like normal software-based PRs.

Reply:
I agree. The config here is mainly used as example of how to use the generic dataset config to load examples. Maybe configure it like the asr_tinyllama?